code/modules/reagents/reagent_containers/medspray.dm

```diff
@@ -0,0 +1,90 @@
+/obj/item/reagent_containers/medspray
+	name = "medical spray"
+	desc = "A medical spray bottle, designed for precision application, with an unscrewable cap."
+	icon = 'icons/obj/chemical.dmi'
+	icon_state = "medspray"
+	item_state = "spraycan"
+	lefthand_file = 'icons/mob/inhands/equipment/hydroponics_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/hydroponics_righthand.dmi'
+	flags_1 = NOBLUDGEON_1
+	obj_flags = UNIQUE_RENAME
+	container_type = OPENCONTAINER
+	slot_flags = SLOT_BELT
+	throwforce = 0
+	w_class = WEIGHT_CLASS_SMALL
+	throw_speed = 3
+	throw_range = 7
+	var/can_fill_from_container = TRUE
+	var/apply_type = PATCH
+	var/apply_method = "spray"
+	var/self_delay = 30
+	var/squirt_mode = 0
+	var/squirt_amount = 5
+	amount_per_transfer_from_this = 10
+	volume = 60
```

Comment:
Move these above the `var/` declarations

Reply:
Done.